### PR TITLE
changed the default backend from wx to qt4

### PR DESCRIPTION
**Pull request by dpinte**

### CODE REVIEW

**Comment by dpinte** [comment]
@rkern here is it

**Comment by corranwebster** [comment]
Can you please update the documentation as well (eg. here https://github.com/enthought/traitsui/blob/master/docs/source/traitsui_user_manual/intro.rst#toolkit-selection)

**Comment by corranwebster** [comment]
And thinking about it, this probably needs to be done in sync with an equivalent change to pyface (eg. https://github.com/enthought/pyface/blob/master/pyface/toolkit.py#L41) or you could have problems with getting menu items and dialogs from the wrong toolkit.

File: CHANGES.txt [hunk 1]
```diff
@@ -4,6 +4,14 @@ Traits UI Changelog
 What's new in Traits 4.6.0
 --------------------------
 
+Potentially breaking change
+
+* Default backend changed from wx to qt4. 
+
+  Application using the wx backend will have to formally set the ETS_TOOLKIT
```
Comment (mdickinson) [inline CHANGES.txt]: `Applications`

File: CHANGES.txt [hunk 2]
```diff
@@ -4,6 +4,14 @@ Traits UI Changelog
...
+  environment variable set to 'wx' if they have the qt4 and wx backend
```
Comment (mdickinson) [inline CHANGES.txt]: extra `set`

File: CHANGES.txt [hunk 3]
```diff
@@ -4,6 +4,14 @@ Traits UI Changelog
 What's new in Traits 4.6.0
```
Comment (mdickinson) [inline CHANGES.txt]: Unrelated to this PR, but why does this say `Traits` rather than `Traits UI`?

**Comment by corranwebster** [comment]
I'm going to merge a bunch of outstanding PRs, since we're moving towards a 4.6 release, and I'd like it to be not entirely trivial.

**Comment by dpinte** [comment]
@corranwebster thanks for the feedback. I'll try to add the missing bits asap but I have a rough agenda for the coming three days. If anyone wants to help, that would be great. 

TODO:

- [ ] update documentation
- [ ] update pyface (ideally we would not need to have multiple places where to set that)

**Comment by corranwebster** [comment]
We had some further discussion on this in the office today.  This still seems a big change to just drop onto people.  @sjagoe suggested a two phase approach to the change:

* A 4.6.0 release (or even 4.5.2?) that issues a deprecation warning when the backend isn't specified.
* A little later, a 5.0 release which makes the actual change.

**Comment by dpinte** [comment]
@corranwebster Can we go on with the two phase approach? We should try to do step 1 asap! Can you connect with the pyface maintainer to synchronize the changes? The idea would be to do the two steps quite quickly (in two weeks e.g.)

**Comment by corranwebster** [comment]
Yes, I think we can.  The pyface maintainer is @rkern, so I don't think he will have any objections, and I can make the appropriate changes in both projects.

**Comment by dpinte** [comment]
Thanks! I think @rkern will probably want to speed things up even faster than a two weeks release!

**Comment by dpinte** [comment]
@corranwebster now that pyface is out with https://github.com/enthought/pyface/pull/150, we should push this one.

**Comment by corranwebster** [comment]
Need to do a deprecate wx release for TraitsUI (was planning to put that together today), but yes, we're underway.

**Pull request merged**